crosslink/cmd/root.go

```diff
@@ -0,0 +1,95 @@
+// Copyright The OpenTelemetry Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package cmd
+
+import (
+	"log"
+	"os"
+
+	"github.com/spf13/cobra"
+	"github.com/spf13/pflag"
+	cl "go.opentelemetry.io/build-tools/crosslink/internal"
+	"go.uber.org/zap"
+)
+
+var (
+	excludeFlags = make([]string, 0)
+	rc           = cl.DefaultRunConfig()
```

Comment:
Using a global for the run configuration has much of this code depending on side effects. It might make sense to bundle the cobra commands and the run config into a struct. That struct would then uniquely scope the command and configuration being run.